src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
Thanks for pointing me to the QString constructor, I'll get onto that soon. In the meantime, is the new quitting logic in 432c2b2 OK?